crates/snp_validation/src/lib.rs

```diff
@@ -39,9 +45,13 @@ flags! {
         SMT = 1 << 0,
     }
 
+    /// These items are mutually exclusive
+    #[derive(Deserialize, Serialize)]
     pub enum PlatformInfoFlags: u8 {
-        TSME = 1 << 1,
-        SMT = 1 << 0,
+        /// Secure Memory Encryption
+        SME = 0,
+        /// Transparent Secure Memory Encryption
+        TSME = 1,
```

Comment:
This style is inconsistent with the enum above. Why was this change made?
Since this is a bit flag, using a decimal is misleading. Either use the bitshift as was done before and common for this kind of stuff, or use explicit bitwise format, like so: https://github.com/rvolosatovs/b/blob/1eeda958ebd4963f28e53ee1789b98d0bf5ded7e/src/lib.rs#L6-L22

Reply:
Fixed